.github/PULL_REQUEST_TEMPLATE/register_did_method.md

```diff
@@ -0,0 +1,47 @@
+## New DID Method Registration
+
+### Required Registration Criteria
+
+Please confirm:
+
+- [ ] The DID Method specification defines the identifer syntax.
+- [ ] The DID Method specification defines the supported DID Operations.
+- [ ] The DID Method specification contains a Privacy and Security Considerations.
+
+### Optional Registration Criteria
+
+Consider ensuring:
+
+- [ ] The registration entry contains contact information.
+- [ ] The registration entry contains registry information.
+
+### Registration Process
+
+In order to register a New DID Method, you must add a json file to [./methods](./methods).
+
+Here is an example registration entry:
+
+```json
+{
+  "status": "registered",
+  "name": "example",
+  "specification": "https://w3c-ccg.github.io/did-spec/",
+  "contact": "W3C Credentials Community Group",
+  "registry": "DID Specification"
+}
+```
+
+Please ensure your pull request and entry are passing the required validation rules.
+
+See the schema requirements for registry entries [here](./tooling/did-method-registry-entry.yml).
+
+### Registration Review Process
+
+Your pull request will be reviewed by the registry editors.
+
+Changes regarging the required criteria may be requested.
+
+If there are no objections or changes requested, 
+
+your method will be registered after a minimum of 7 days.
+
```

Comment:
I don't think this needs to be multiple paragraphs... For certain, the last two lines of text should be one.
Also, fixed "regarging" typo
```suggestion
Your pull request will be reviewed by the registry editors.
Changes regarding the required criteria may be requested.
If there are no objections or changes requested, 
your method will be registered after a minimum of 7 days.
```